Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -110,14 +116,24 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
         
         messageComposerViewController.view.translatesAutoresizingMaskIntoConstraints = false
         addChildViewController(messageComposerViewController, targetView: view)
-
+        
         messageComposerViewController.view.topAnchor.pin(equalTo: collectionView.bottomAnchor).isActive = true
         messageComposerViewController.view.leadingAnchor.pin(equalTo: view.safeAreaLayoutGuide.leadingAnchor).isActive = true
         messageComposerViewController.view.trailingAnchor.pin(equalTo: view.safeAreaLayoutGuide.trailingAnchor).isActive = true
         messageComposerBottomConstraint = messageComposerViewController.view.bottomAnchor.pin(equalTo: view.bottomAnchor)
         messageComposerBottomConstraint?.isActive = true
+        
+        if channelController.channel?.config.typingEventsEnabled ?? false {
+            view.addSubview(typingIndicatorView)
+            typingIndicatorView.heightAnchor.pin(equalToConstant: 22).isActive = true
+            typingIndicatorView.pin(anchors: [.leading, .trailing], to: view)
+            typingIndicatorView.bottomAnchor.pin(equalTo: messageComposerViewController.view.topAnchor).isActive = true
+            typingIndicatorView.isHidden = true
+            
+            collectionView.contentInset.bottom = 22
```

Comment:
nit a) would be nice to have the `22` abstracted into a local variable when it's used twice
nit b) you shouldn't override the existing value but rather do something like `+= 22`

Reply:
Totally right, changed.